src/tests/topp/QCCalculator_2_output.mzQC

```diff
@@ -79,21 +79,39 @@
             "name": "Total ion current chromatogram",
             "value": {
               "Relative intensity": [
-                0,
-                120,
-                120
+                0.0,
+                100.0,
+                100.0,
+                91.66666412353516
               ],
               "Retention time": [
                 -1.0,
```

Comment:
is -1 a valid value?

Reply:
in this mzML test file there is a RT of -1.0 yes